### PR TITLE
test: automated cases for smoke tcs tc1622 tс1565 tc1674 tc1597

### PR DESCRIPTION
**Pull request by amelnytskyi**
Added autotests fot the next cases
TC1674 Withdraw - Artifacts - "Insufficient balance" warning message (Zero token balance) 
TC1622 Withdraw - Sent to - Redirection
TC1565 Withdraw - Confirm transaction modal - Redirection 
TC1597 Bridge - Withdraw - Artifacts (completed active partly manual tc)

### CODE REVIEW

File: tests/e2e/features/artifacts/assetsPage/artifacts-emptyWallet.feature [hunk 1]
```diff
@@ -46,3 +46,15 @@ Feature: Artifacts - UI
...
+  Scenario: Check "Insufficient balance" warning message (Zero token balance) (Withdraw)
+    Then A wallet should be "empty"
+    When I go to page "/transaction/zksync/era/withdraw"
+    When I click by "text" with "Your account" value
+    When I choose "ETH" as token and insert "1000" as amount
+    Then Message " Insufficient balance " should be visible
+    #Error state
+    Then Element with "partial class" "has-error" should be "visible"
+    Then Element with "partial class" "amount-input-error" should be "visible"
+
```
Comment (pcheremu) [inline tests/e2e/features/artifacts/assetsPage/artifacts-emptyWallet.feature:60]: step 1.3 is missed https://allure.matterlabs.dev/project/17/test-cases/1674?treeId=30

File: tests/e2e/features/artifacts/bridgePage/artifacts.feature [hunk 2]
```diff
@@ -102,6 +102,9 @@ Feature: Withdraw
     Given I go to page "/bridge?network=era-goerli"
     When I click by text "Withdraw"
     Then Element with "text" "Bridge" should be "visible"
+    Then Element with "partial text" "Recent withdrawals" should be "visible"
```
Comment (pcheremu) [inline tests/e2e/features/artifacts/bridgePage/artifacts.feature:105]: Is "Number of tx in progress" missed? https://allure.matterlabs.dev/project/17/test-cases/1597?treeId=30 And what about other steps, it is hart now to have a parallel between steps in the code and tests. Can you please align or group steps at least?

File: tests/e2e/features/redirection/mainPage/redirection.feature [hunk 3]
```diff
@@ -306,4 +306,26 @@ Feature: External Redirection on the Main Page
     Then Message "Transaction submitted" should be visible
     When I click by text " Explore ecosystem "
     Then New page has "https://ecosystem.zksync.io/" address
+
+  @id1622
```
Comment (pcheremu) [inline tests/e2e/features/redirection/mainPage/redirection.feature:310]: Please add this steps to Allure https://allure.matterlabs.dev/project/17/test-cases/1622?treeId=30

File: tests/e2e/features/redirection/mainPage/redirection.feature [hunk 4]
```diff
@@ -306,4 +306,26 @@ Feature: External Redirection on the Main Page
...
+  Scenario: Check redirection for the "Send to" page (Withdraw)
+    Given I go to page "/transaction/zksync/era/withdraw"
+    When I click by "text" with "Your account" value
+    When I confirm the network switching
+    Then Element with "text" " Arriving in ~24 hours " should be "visible"
+    When I click by "text" with " Arriving in ~24 hours " value
+    Then Element with "text" " Arriving in ~24 hours " should be "clickable"
+    Then New page has "https://era.zksync.io/docs/reference/troubleshooting/withdrawal-delay.html#withdrawal-delay" address
+    
+  @id1565
```
Comment (pcheremu) [inline tests/e2e/features/redirection/mainPage/redirection.feature:320]: the same as above, no info in Allure

Comment (pcheremu) [reply tests/e2e/features/redirection/mainPage/redirection.feature:320]: https://allure.matterlabs.dev/project/17/test-cases/1565?treeId=30

**Review by pcheremu — CHANGES_REQUESTED** [review]

File: tests/e2e/features/artifacts/bridgePage/artifacts.feature (hunk 2 above)
Comment (amelnytskyi) [reply tests/e2e/features/artifacts/bridgePage/artifacts.feature:105]: To add check for "Recent withdraw" counter to Withdraw tc

File: tests/e2e/features/redirection/mainPage/redirection.feature (hunk 3 above)
Comment (amelnytskyi) [reply tests/e2e/features/redirection/mainPage/redirection.feature:310]: Preconditions added in Allure

File: tests/e2e/features/redirection/mainPage/redirection.feature (hunk 4 above)
Comment (amelnytskyi) [reply tests/e2e/features/redirection/mainPage/redirection.feature:320]: Preconditions added in Allure

**Review by pcheremu — APPROVED** [review]

**Pull request merged**